src/cmdlib.sh

```diff
@@ -307,7 +307,7 @@ EOF
 
     # 'pci' bus doesn't work on aarch64
     pcibus=pci.0
-    if [ "$(arch)" = "aarch64" ]; then
```

Comment:
this technically works, right? http://mywiki.wooledge.org/BashFAQ/031

Reply:
Ah sorry - I thought it HAD helped but i was wrong.
Closing it out.